buildroot/share/PlatformIO/boards/genericSTM32F407VGT6.json

```diff
@@ -0,0 +1,56 @@
+{
+  "build": {
+    "core": "stm32",
+    "cpu": "cortex-m4",
+    "extra_flags": "-DSTM32F407xx -DSTM32F4",
+    "f_cpu": "168000000L",
+    "hwids": [
+      [
+        "0x1EAF",
+        "0x0003"
+      ],
+      [
+        "0x0483",
+        "0x3748"
+      ]
+    ],
+    "mcu": "stm32f407vgt6",
+    "product_line": "STM32F407xx",
+    "variant": "Generic_F4x7Vx"
+  },
+  "debug": {
+    "default_tools": [
+      "stlink"
+    ],
+    "jlink_device": "STM32F407VG",
+    "openocd_extra_args": [
+      "-c",
+      "reset_config none"
+    ],
+    "openocd_target": "stm32f4x",
+    "svd_path": "STM32F40x.svd"
+  },
+  "frameworks": [
+    "arduino",
+    "cmsis",
+    "stm32cube",
+    "libopencm3"
+  ],
+  "name": "STM32F407VG (192k RAM. 1024k Flash)",
+  "upload": {
+    "disable_flushing": false,
+    "maximum_ram_size": 131072,
```

Comment:
Some json parsers don't allow comments inside the file. That's the reason a did a separated commit when adding this file with a message about wrong ram. 
I will check if others pio json files have comments, but I would not put comments on it.

Reply:
Comments are non-compliant JSON.  I meant you could just change the name to something like this:

STM32F407VG (128k RAM, 64k CCM RAM, 1024k Flash)

That way the name still reflects 192k, but it is clear that 64k is different.

This isn’t very important. I’m just trying to prevent someone breaking it again by trying to “correct” the mismatch.